### PR TITLE
[DOCS] Reformat get field mapping API docs

### PR DESCRIPTION
**Pull request by jrodewig**
This PR updates the get field mapping API to align with the new [API reference template](https://github.com/elastic/docs/blob/master/shared/api-ref-ex.asciidoc).

Relates to elastic/docs#937 and #43765

Dependent on #45699.

### CODE REVIEW

**Comment by jrodewig** [comment]
CI failures are expected until #45699 is merged.

**Comment by jrodewig** [comment]
@elasticmachine update branch

**Comment by jrodewig** [comment]
@elasticmachine update branch

File: docs/reference/indices/get-field-mapping.asciidoc [hunk 1]
```diff
@@ -167,11 +202,24 @@ returns:
...
-[float]
-==== Other options
+[[get-field-mapping-api-multi-index-ex]]
+===== Multiple indices and fields
 
-[horizontal]
-`include_defaults`::
+The get field mapping API can be used to get the mapping of multiple fields from more than one index
+with a single call. General usage of the API follows the
+following syntax: `host:port/{index}/_mapping/field/{field}` where
```
Comment (szabosteve) [inline docs/reference/indices/get-field-mapping.asciidoc]: Maybe we could change "{var}" to "< var >" in line 210-212, too.

Comment (szabosteve) [inline docs/reference/indices/get-field-mapping.asciidoc]:
```suggestion
following syntax: `host:port/<index>/_mapping/field/<field>` where
```

File: docs/reference/indices/get-field-mapping.asciidoc [hunk 2]
```diff
@@ -167,11 +202,24 @@ returns:
...
+`{index}` and `{field}` can stand for comma-separated list of names or wild cards. To
```
Comment (szabosteve) [inline docs/reference/indices/get-field-mapping.asciidoc]:
```suggestion
`<index>` and `<field>` can stand for comma-separated list of names or wild cards. To
```

File: docs/reference/indices/get-field-mapping.asciidoc [hunk 3]
```diff
@@ -167,11 +202,24 @@ returns:
...
+get mappings for all indices you can use `_all` for `{index}`. The
```
Comment (szabosteve) [inline docs/reference/indices/get-field-mapping.asciidoc]:
```suggestion
get mappings for all indices you can use `_all` for `<index>`. The
```

**Review by szabosteve — CHANGES_REQUESTED** [review]
I think there might be some `{var}` in the piece that should be changed to `<var>`.

**Comment by jrodewig** [comment]
Good catches, @szabosteve!

**Review by szabosteve — APPROVED** [review]
Thanks!

LGTM!

**Pull request merged**